From 1022e92116781ba5b44cccacb4767351b5d47907 Mon Sep 17 00:00:00 2001
From: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Date: Wed, 17 May 2023 15:58:59 +0200
Subject: [PATCH 523/529] ARM: dts: stm32: move linux,cma node for
 stm32mp157[a|d]-ev1 boards

Move linux,cma node from stm32mp157a-ev1-scmi.dtsi to
stm32mp157[a|d]-ed1.dts for stm32mp157[a|d]-ev1 boards

Change-Id: I9eb065ba33904ae535cec1fd568de74d510f5c29
Signed-off-by: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/307013
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm/boot/dts/stm32mp157a-ed1.dts       | 9 +++++++++
 arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi | 9 ---------
 arch/arm/boot/dts/stm32mp157d-ed1.dts       | 9 +++++++++
 3 files changed, 18 insertions(+), 9 deletions(-)

--- a/arch/arm/boot/dts/stm32mp157a-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157a-ed1.dts
@@ -83,6 +83,15 @@
 			reg = <0xe8000000 0x8000000>;
 			no-map;
 		};
+
+		/* global autoconfigured region for contiguous allocations */
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x8000000>;
+			alignment = <0x2000>;
+			linux,cma-default;
+		};
 	};
 
 	led {
--- a/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
+++ b/arch/arm/boot/dts/stm32mp157a-ev1-scmi.dtsi
@@ -10,15 +10,6 @@
 			reg = <0xfe000000 0x2000000>;
 			no-map;
 		};
-
-		/* global autoconfigured region for contiguous allocations */
-		linux,cma {
-			compatible = "shared-dma-pool";
-			reusable;
-			size = <0x8000000>;
-			alignment = <0x2000>;
-			linux,cma-default;
-		};
 	};
 };
 
--- a/arch/arm/boot/dts/stm32mp157d-ed1.dts
+++ b/arch/arm/boot/dts/stm32mp157d-ed1.dts
@@ -83,6 +83,15 @@
 			reg = <0xe8000000 0x8000000>;
 			no-map;
 		};
+
+		/* global autoconfigured region for contiguous allocations */
+		linux,cma {
+			compatible = "shared-dma-pool";
+			reusable;
+			size = <0x8000000>;
+			alignment = <0x2000>;
+			linux,cma-default;
+		};
 	};
 
 	led {
